spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
```

Comment:
If you put rspec descriptions into reopening the classes, you unintentionally make the potential private constants available to the examples. This can easily end up in testing against implementation details, not the public API.

For these, and the reason to remove indentation I suggest to kill this re-opening for all rspec tests.

I realize the whole code base had to be changed and this is out of scope for this PR. Commenting to start the discussion for a followup PR that fixes the entire class.